package.json

```
@@ -44,7 +44,7 @@
     "dev": "apps/shared/test-helpers/develop-kit.sh",
     "devchain:reset": "aragon devchain --reset",
     "devchain": "lerna run devchain --stream",
-    "frontend": "lerna run frontend --stream --parallel",
+    "frontend": "lerna run frontend --stream",
```

Yeah, let's re-add the parallel flag for a boost, but also let's remove `--stream` that seems redundant when using with `--parallel` (reference: https://github.com/lerna/lerna/tree/master/commands/run#--parallel))

```suggestion
    "frontend": "lerna run frontend --parallel",
```

---

Strange that you saw a difference with this! It doesn't seem like it should affect it at all, to me, from what I understand of lerna. But it also won't hurt anything, and it seems like it might actually save some time – though I do wonder if you were just seeing some statistical noise, there.

---

Yeah, it is just for cleanup purposes, as per lerna documentation `--parallel` flag already enables `--stream` so no need to add that flag again :)